ui/src/main/webapp/index.html

```diff
@@ -67,7 +70,7 @@
             packageConfigPaths: ['node_modules/*/package.json']
         });
     </script>
-    <script src="node_modules/rxjs/bundles/Rx.umd.js"></script>
+    <script src="node_modules/rxjs/bundles/Rx.min.js"></script>
```

Comment:
I am still far from an expert on JavaScript module systems, however from what I understand the answer to your question is yes. My understanding is that the "umd" version is designed primarily to work with the "umd" module system. Because we are using the "SystemJS" module system, I think it is generally preferred not to use the umd builds.

If you have information to the contrary, let me know of course. :)